akka-http-core/src/main/scala/akka/http/impl/engine/ws/Handshake.scala

```diff
@@ -22,7 +28,8 @@ import scala.reflect.ClassTag
 private[http] object Handshake {
   val CurrentWebsocketVersion = 13
```

Comment:
Glad that I asked then! 
Yes, let's please add a test case for "client wants lower version", it's pretty simple as we'll simply reply with what we have instead (13)

refs https://tools.ietf.org/html/rfc6455#section-4.4